modules/home/assets/js/icons/side-bar-check-icon.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { SvgIcon } from '@elementor/ui';
+import { PropTypes } from 'prop-types';
+const SideBarCheckIcon = ( props ) => {
+	return (
+		<SvgIcon viewBox="0 0 25 25" { ...props }>
```

Comment:
Generally speaking, our sizes should be multiples of 4 or 8px. Can you please check that with Hadar?

Reply:
checking